apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Snippets/StorageSnippetFixture.cs

```diff
@@ -63,6 +68,43 @@ private void CreateAndPopulateBucket()
             RegisterLocalFileToDelete(WorldLocalFileName);
         }
 
+        private void CreateNotificationsBucket()
+        {
+            string bucket = "snippets-notifications-" + Guid.NewGuid().ToString().ToLowerInvariant();
+            var topicName = new TopicName(ProjectId, bucket);
+            var storageClient = StorageClient.Create();
+            storageClient.CreateBucket(ProjectId, bucket);
+        }
+
+        internal Notification CreateNotification(string prefix)
+        {
+            var publisherClient = PublisherClient.Create();
+            var topicName = new TopicName(ProjectId, "topic-" + Guid.NewGuid().ToString().ToLowerInvariant());
+            publisherClient.CreateTopic(topicName);
+            RegisterTopicToDelete(topicName);
+            var storageClient = StorageClient.Create();
+            string storageServiceAccount = $"serviceAccount:{storageClient.GetStorageServiceAccountEmail(ProjectId)}";
+
+            // TODO: Simplify this when we have IAM convenience methods.
```

Comment:
This TODO can now be done?

Reply:
Well, once we've got a release of IAM. Let's talk about that later.